### PR TITLE
New package: sbt-0.13.13

### PR DESCRIPTION
**Pull request by felher**
Hello folks :)

I added the scala built tool (sbt), mainly for scala programmers (although it is used with other languages as well)
A xlint was done and the only message I got was

> srcpkgs/sbt/template:11: custom variables should use _ prefix: XBPS_FETCH_CMD=wget

But as far as I know, XBPS_FETCH_CMD is a official variable.
Of course I tested it locally and it worked fine.

This is the first package I created. If there is anything wrong with it, please let me know.

### CODE REVIEW

File: srcpkgs/sbt/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+license="BSD-3-Clause"
+homepage="http://www.scala-sbt.org/index.html"
+distfiles="http://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/${version}/sbt-launch.jar"
+checksum=9cd7d56eabfb15b5fe68c39116c4cfe6379b5b260d63086d4f5c69fa62521c13
+
+do_install() {
+	vmkdir usr/lib/sbt
+	vcopy sbt-launch.jar usr/lib/sbt/
+
+	vlicense ${FILESDIR}/LICENSE BSD-3-Clause
```
Comment (chneukirchen) [inline srcpkgs/sbt/template]: Don't rename the license file.

File: srcpkgs/sbt/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#the default downloader always give 'permission denied' when trying to download
+#the distfile
+hostmakedepends="wget"
+XBPS_FETCH_CMD=wget
+
+depends="virtual?java-runtime"
+short_desc="A build tool for Scala, Java, and more"
+maintainer="Felix Herrmann <felix@herrmann-koenigsberg.de>"
+license="BSD-3-Clause"
```
Comment (chneukirchen) [inline srcpkgs/sbt/template]: Use `3-clause-BSD`.

Comment (felher) [inline srcpkgs/sbt/template]: > license A string matching any license file available in /usr/share/licenses. Multiple licenses should be separated by commas, i.e GPL-3, LGPL-2.1.

Neither mine nor your string matches a file in that directory. I just took the name github gave the license. Why should I choose "3-clause-BSD" instead and is it a problem that it doesn't match an existing file?

File: srcpkgs/sbt/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+homepage="http://www.scala-sbt.org/index.html"
+distfiles="http://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/${version}/sbt-launch.jar"
+checksum=9cd7d56eabfb15b5fe68c39116c4cfe6379b5b260d63086d4f5c69fa62521c13
+
+do_install() {
+	vmkdir usr/lib/sbt
+	vcopy sbt-launch.jar usr/lib/sbt/
+
+	vlicense ${FILESDIR}/LICENSE BSD-3-Clause
+	vinstall ${FILESDIR}/sbt 655 /usr/bin/
```
Comment (felher) [inline srcpkgs/sbt/template]: Right, will fix.

File: srcpkgs/sbt/template (hunk 1 above)
Comment (felher) [inline srcpkgs/sbt/template]: Thanks, will fix

File: srcpkgs/sbt/template (hunk 2 above)
Comment (pullmoll) [inline srcpkgs/sbt/template]: `3-clause-BSD` is the Void Linux choice for the name of the license. This has nothing to do with the actual filename of the license installed with `base-files`, but is merely a standardization of templates. 240 other templates currently have `license="3-clause-BSD"`

Comment (felher) [inline srcpkgs/sbt/template]: Oh, yeah. I should have thought of checking the other packages first. Thanks for the explanation, will fix.

**Comment by felher** [comment]
Okay, I think I fixed my mistakes. Anything more that needs fixing? :)

**Comment by pullmoll** [comment]
Please squash the commits into one using the first commit message.

**Comment by felher** [comment]
Okay, squashed it. Since I had to push -f anyway I also took the liberty to do a fresh rebase

**Comment by felher** [comment]
Any news?

**Pull request closed without merging**